pandas/_libs/tslibs/period.pyx

```diff
@@ -17,26 +17,26 @@ from pandas.compat import PY2
 
 cimport cython
 
-from tslibs.np_datetime cimport (pandas_datetimestruct,
-                                 dtstruct_to_dt64, dt64_to_dtstruct,
-                                 is_leapyear)
+from pandas._libs.tslibs.np_datetime cimport (pandas_datetimestruct,
```

Comment:
the beauty here is that you can do

``from np_datetime cimport ....`` and so on , the puts ``period`` in the same dir as all of the tslibs